### ISSUE TITLE
[a11y] Issue 22: Missing first-level heading (Closed Page)

### ISSUE THREAD

**Issue by NewJerseyStyle** [NewJerseyStyle/polis#80]
**Description:**

The closed page is missing a first-level heading (`<h1>`).

**Steps to Reproduce:**

1. Navigate to the closed page.
2. Inspect the page's heading structure.

**Expected Behavior:**

A `<h1>` element should be present.

**Actual Behavior:**

A `<h1>` element is missing.

**Component/Area:** Closed Page

**Source:** Auto evaluation

**Affected Elements:** `<h1>`

**Comment by yshng** [comment]
I was able to reproduce this today running a local instance using Docker
![h2-closed](https://github.com/user-attachments/assets/16020820-badd-4828-b57a-b89f05b7da07)

However, the current [code](https://github.com/CivicTechTO/polis/blob/3df237827db0e4d9ff5eca8b3a308986cd875846/client-participation/js/templates/participation.handlebars#L171) in edge-civittechto shows that the `<h2>` was already replaced by `<h1>` in [#32](https://github.com/CivicTechTO/polis/pull/32) by @thomassth on Oct 17 - This issue was opened Oct 30. Perhaps I don't understand something about Docker deployment, or what branch the a11y testing was conducted on? Otherwise it seems this is no longer an issue and can be closed.

**Comment by NewJerseyStyle** [comment]
No problem. The issue was logged earlier in Google Sheet and later moved to GitHub issues which caused the PR occur before the issue is created 🚀 I guess Tom found the issue in Google Sheet and fixed it in #32 before we sync everything with GitHub (Google Sheet is always give you the fresh a11y issue, GitHub issues is slower)

Thank you @yshng for pointing that out. Let's close this issue since it's been fixed
